### ISSUE TITLE
Extract SPARQLWrapper code into an importable code snippet

### ISSUE THREAD

**Issue by andrewtavis** [scribe-org/Scribe-Data#164]
### Terms

- [X] I have searched [open and closed feature requests](https://github.com/scribe-org/Scribe-Data/issues?q=is%3Aissue+label%3Afeature)
- [X] I agree to follow Scribe-Data's [Code of Conduct](https://github.com/scribe-org/Scribe-Data/blob/main/.github/CODE_OF_CONDUCT.md)

### Description

Something that would be great to get to would be to extract the SPARQLWrapper logic into a separate function that can be imported. Specifically, in multiple files we have the following:

```py
from SPARQLWrapper import SPARQLWrapper
endpoint_url = "https://query.wikidata.org/sparql"
sparql = SPARQLWrapper(endpoint_url)
```

Maybe we can put all of this into a `wikidata_utils.py` file in [src/scribe_data/wikidata](https://github.com/scribe-org/Scribe-Data/tree/main/src/scribe_data/wikidata) 🤔

### Contribution

Happy to support someone who has interest in working on this, or can also get to it myself at some point 😊

**Comment by andrewtavis** [comment]
Ping @mhmohona 👋 We don't necessarily have to have this be a part of GSoC. You'd be welcome to work on this if you'd like to though :)

**Comment by axif0** [comment]
Hey @andrewtavis , I want to work on this issue.

**Comment by andrewtavis** [comment]
Sure thing, @axif0! Let us know if there's anything we can do to support :)

**Comment by axif0** [comment]
PR is on the way 😎

**Comment by andrewtavis** [comment]
Closed via #166 :) Thanks for the quick work here, @axif0! Let us know if other issues sound appealing!
